ui.tests/test-module/specs/wizard/wizard.authoring.spec.js

```diff
@@ -87,23 +87,23 @@ describe('Page - Authoring', function () {
             cy.deleteComponentByPath(wizardLayoutDrop) ;
         });
 
-        // todo: flaky
-        it('verify Navigation Working between tabs in Authoring', { retries: 3 }, function(){
+        it('verify Navigation Working between tabs in Authoring', function(){
```

Comment:
@rismehta / @deepprakash345 I have removed retries and moved the logic to execute post wait.